xml/System/Memory`1.xml

```diff
@@ -43,9 +68,17 @@
         <Parameter Name="array" Type="T[]" />
       </Parameters>
       <Docs>
-        <param name="array">To be added.</param>
-        <summary>To be added.</summary>
-        <remarks>To be added.</remarks>
+        <param name="array">The array from which to create the <see cref="T:System.Memory`1" /> object.</param>
+        <summary>Creates a new <see cref="T:System.Memory`1" /> object over the entirety of a specified array.</summary>
+       <remarks>
+          <format type="text/markdown"><![CDATA[  
+
+If `array` is `null`, this constructor returns a `null` <xref:System.Memory%601> object.
+
+      ]]></format>
+        </remarks>
+        <exception cref="T:System.ArrayTypeMismatchException">
+          <paramref name="T" /> is a reference type, and <paramref name="array" /> is not an array of type <paramref name="T" />.</exception>
```

Comment:
Would it be useful to mention that this could occur due to co-variant arrays?
https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/concepts/covariance-contravariance/